### PR TITLE
update DigiAssets Learn the protocol. #57 link

### PR DESCRIPTION
**Pull request by DennisPitallano**
update DigiAssets Learn the protocol. #57 link

### CODE REVIEW

File: fil/index.html
Comment (DGBNOOB) [inline fil/index.html:1]: Line 757 has the old URL to https://github.com/DigiByte-Core/DigiAssets-Protocol-Specifications/wiki. 

Simple cut and paste mistake.

the correct URL is https://github.com/DigiAsset-Core/DigiAsset_Core/tree/master/web

Comment (DennisPitallano) [reply fil/index.html:1]: nice catch.

Comment (DennisPitallano) [reply fil/index.html:1]: already resolved

Comment (DGBNOOB) [reply fil/index.html:1]: Line 757 has been resolved.

**Review by DGBNOOB — APPROVED** [review]
Commits c729a68
update DigiAssets Learn the protocol

These changes address issue #57 DigiAssets “Learn the protocol”. Issue 57 addresses the URL link in the DigiAssets section of DigiByte .org, “Learn the protocol”. It currently directs the viewer to the v1-v2 DigiAssets specification on GitHub and this change is to update the website DigiAssets info to v3 DigiAssets/DigiAsset Core repository.
The purpose for this update is to give developers updated info to build with the v3 DigiAsset protocol and DigiAsset-Core implementation.
Instead of one link, there will be two; the first directs the viewer to DigiAsset Core/Web/ and the second link directs the viewer to DigiAsset Core src/rpc/methods/

There were 37 files changed, 111 additions and 37 deletions. In the English version you can see the additions: lines 787, 788 and 789. Index.html
36 language versions to include English and (en-us). Some changes were translated to native language version.
I reviewed the changes, I visited the 2 URL links, however, I did not test the changes. I recommend it for approval.

Dennis,
Thank you again, great job.
cACK

**Review by ycagel — APPROVED** [review]
cACK

**Pull request merged**